src/player.js

```diff
@@ -170,12 +171,19 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  /**
+   * The event manager of the current media.
+   * @type {EventManager}
+   * @private
+   */
+  _mediaEventManager: EventManager;
   /**
    * The event manager of the player.
    * @type {EventManager}
    * @private
    */
-  _eventManager: EventManager;
+  _playerEventManager: EventManager;
+
```

Comment:
why two seperate event managers?

Reply:
The current event manager we have is cleaned and attached every media switch.
We need an event manager that is kept through all the player instance.

Reply:
So see comment about separating to its own manager, we do same in other managers.

Reply:
Not relevant anymore. Reverted back to one manager